src/Modules/UngradedModules.jl

```diff
@@ -925,10 +929,9 @@ SubModuleOfFreeModule(F::FreeMod{L}, A::MatElem{L}) where {L} = SubModuleOfFreeM
 Construct the submodule generated by the rows of `A`.
 
 !!! note
-
-    The embedding free module of the submodule is constructed by the function and therefore
-    not compatible with free modules that are defined by the user or other functions. 
-    If you need compatibility use `SubModuleOfFreeModule(F::FreeMod{L}, A::MatElem{L}) where {L}`.
+    The ambient free module of the submodule is constructed by the function and therefore
+    not compatible with free modules that are defined by the user or by other functions. 
```

Comment:
Honest question, what does "compatible" mean in this context?

Reply:
Good question: If still needed, I will make this more precise when working on module.md (I am still busy with a new docu file called free_modules.md). So this should not hold up the merge.

Reply:
This means for example that `SubQuo(A,B) == SubQuo(A,B)` will give an error because the ambient free modules are different (not compatible).